### DISCUSSION TITLE
A very small area but it takes a long time to unwrap

### DISCUSSION THREAD

**Discussion by FAN-CHENG-peng** [alexisInSAR/EZ-InSAR#27] [Help]
ISCE unwrapping takes too time. Is there any way to speed up the processing?

**Comment by alexisInSAR** [comment]
Hi, 

There are several possibilities to speed up the unwrapping processing (multilooking, masking, ...). Can you show an interferogram to get a better idea?

**Reply by FAN-CHENG-peng** [reply]
Thank you very much for your reply,
Although I chose a very small ROI, I found that ez-insar was unwrapping all the bursts containing the ROI, resulting in a very slow processing.
Can we unwrap only the ROI?

**Reply by alexisInSAR** [reply]
Indeed, during a MintPy-based processing, the unwrapping is done via ISCE and before cropping of images. This can be tricky for your example as it is not optimised. 

In the current version, it is not possible to unwrap only the ROI under MintPy (why not in a next version). However, this option is available in the StaMPS-based processing.
